lib/realtime/application.ex

```diff
@@ -62,7 +67,8 @@ defmodule Realtime.Application do
         RealtimeWeb.Presence,
         {Task.Supervisor, name: Realtime.TaskSupervisor},
         Realtime.Latency,
-        Realtime.Telemetry.Logger
+        Realtime.Telemetry.Logger,
+        Realtime.Tenants.Check
```

Comment:
This needs to be like a DynamicSupervisor where you start a Check on the correct node for a tenant.

And you can start is with a `name` `via` such that it registers with :syn.

Then if Check is alive for a tenant and we have it in syn then we know the db is okay.

Reply:
Yes, will add the clustering later, just getting the childspec make sense